ceph-iscsi-config-flake8/build/build

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e
+set -x
+
+function generate_filelist(){
+   if [[ "$pull_request_id" -eq "" || "${ghprbCommentBody:-}" = "jenkins flake8 all" ]]
+   then
+       find . -name '*.py'
+   else
+       curl -XGET "https://api.github.com/repos/ceph/ceph-iscsi-config/pulls/$pull_request_id/files" |
+       jq '.[].filename' |  # just the files please
+       tr -d '"' |  # remove the quoting from JSON
+       grep ".py$" # just the python
+   fi
+
+}
+
+function check(){
+    local file
+    while read -r filename; do
+        pushd "$(dirname "$filename")"
+        file=$(basename "$filename")
+        sudo docker run --rm -v "$workspace"/ceph-iscsi-config/tox.ini:/tox.ini -v "$(pwd)"/"$file":/"$file" eeacms/flake8 /"$file"
```

Comment:
why is docker required to run flake8 here?

Reply:
No clue -- I copied from another flake8 checker

Reply:
... and I'd imagine this will eventually need docker for future integration tests

Reply:
@dillaman https://stickler-ci.com/ is an alternative for running flake8 on Python projects. I've used it for a couple projects on Github, and it was really easy to set up.